src/api/time/IndependentTimeContext.js

```diff
@@ -226,10 +226,12 @@ class IndependentTimeContext extends TimeContext {
     const previousClock = this.activeClock;
     if (previousClock) {
       previousClock.off('tick', this.tick);
+      previousClock.stop();
     }
 
     this.activeClock = clock;
     this.activeClock.on('tick', this.tick);
+    this.activeClock.start();
```

Comment:
Yeah on reflection clocks shouldn't be stopped or started because we'll be sharing instances. Assume they will sort themselves out.